engines/scumm/input.cpp

```diff
@@ -182,6 +197,8 @@ void ScummEngine::parseEvent(Common::Event event) {
 			// Fixes bug #3173: "FT: CAPSLOCK + V enables cheating for all fights"
 			//
 			// Fingolfin remarks: This wouldn't be a problem if we used keycodes.
+			//
+			// TODO: Is this still needed now that INSANE uses keymapper actions?
 			_keyDownMap[toupper(event.kbd.ascii)] = false;
```

Comment:
The original allows multiple keys to be pressed.  So, perhaps, it is still needed.